projects/demo/src/modules/components/range/index.html

```diff
@@ -179,6 +167,24 @@
                     .
                 </p>
             </tr>
+            <tbody>
+                <tr>
+                    <td colspan="3">
+                        <span tuiTitle>
+                            <strong>CSS customization</strong>
+                        </span>
+                    </td>
+                </tr>
+
+                <tr
+                    name="[style.--tui-thumb-radius.rem]"
```

Comment:
I hope we should use `px` here (only for this part of documentation & `Slider` documentation)

```suggestion
                    name="[style.--tui-thumb-radius.px]"
```

**Still use `rem` inside library code!**

___

https://github.com/user-attachments/assets/b741e69c-f08c-410f-b9b9-ac577b9bb8c0

vs

https://github.com/user-attachments/assets/2863dd4b-cff4-4647-8efa-4654d6491e45

More smooth increment